### PR TITLE
Updating README.md with more complete steps

### PR DESCRIPTION
**Pull request by donmccasland**
Look at rendered update here:
https://github.com/donmccasland/jenkins-integration-samples/tree/master/gke

### CODE REVIEW

**Comment by craigdbarber** [comment]
Could you please revert the mv of the Jenkinsfile. It's meant to be in the gke sub-project folder. You can point your multibranch pipeline project to it by specifying: "gke/Jenkinsfile" as the Jenkins file path.

File: gke/README.md [hunk 1]
```diff
@@ -51,8 +51,36 @@ The following describes the setup process for this project:
 1. Install Jenkins on GKE using the stable/jenkins helm chart: [Jenkins On GKE Tutorial](
    https://cloud.google.com/solutions/jenkins-on-kubernetes-engine-tutorial).
 
-1. Install and configure the GKE plugin: [Jenkins GKE Docs](
-   https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md).
+1. Install and configure the following plugins:
+   1. GKE plugin: [Jenkins GKE Docs](https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md).
+   1. GCS plugin: [Jenkins GCS Docs](https://github.com/jenkinsci/google-storage-plugin/blob/develop/README.md).
+   1. Global Variable String Parameter plugin: [Jenkins Global Var Docs](https://wiki.jenkins.io/display/JENKINS/Global+Variable+String+Parameter+Plugin).
```
Comment (craigdbarber) [inline gke/README.md]: The global var param plugin isn't used in any of the downstream steps. Adding it as a requirement seems unnecessary.

File: gke/README.md [hunk 2]
```diff
@@ -51,8 +51,36 @@ The following describes the setup process for this project:
...
+   1. Blue Ocean: [Blue Ocean Docs](https://jenkins.io/doc/book/blueocean/getting-started/#on-an-existing-jenkins-instance).
```
Comment (craigdbarber) [inline gke/README.md]: Similar to above, the Blue Ocean plugin isn't required for this example. If customers want to use it that's fine, but I don't want to make it seem like it's required for this project to function properly.

File: gke/README.md (hunk 1 above)
Comment (donmccasland) [reply gke/README.md]: I believe you need it for step 6, adding global parameters.  Can you check your installation?  If you don't have, let's remove this step.

File: gke/README.md (hunk 2 above)
Comment (donmccasland) [reply gke/README.md]: That's fine, and I'd love us to simplify actually.  Previously the README linked to https://jenkins.io/doc/tutorials/build-a-multibranch-pipeline-project/ which requires it.

File: gke/README.md (hunk 1 above)
Comment (craigdbarber) [reply gke/README.md]: My installation does not have it.

File: gke/README.md (hunk 2 above)
Comment (craigdbarber) [reply gke/README.md]: I didn't realize they did, I can see how that would be misleading. But confirmed, Blue Ocean is not required.

Comment (donmccasland) [reply gke/README.md]: Okay, let uninstall and rewrite these steps a little bit.

File: gke/README.md [hunk 3]
```diff
@@ -51,8 +51,34 @@ The following describes the setup process for this project:
 1. Install Jenkins on GKE using the stable/jenkins helm chart: [Jenkins On GKE Tutorial](
    https://cloud.google.com/solutions/jenkins-on-kubernetes-engine-tutorial).
 
-1. Install and configure the GKE plugin: [Jenkins GKE Docs](
-   https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md).
+1. Install and configure the following plugins:
+   1. GKE plugin: [Jenkins GKE Docs](https://github.com/jenkinsci/google-kubernetes-engine-plugin/blob/develop/docs/Home.md).
+   1. GCS plugin: [Jenkins GCS Docs](https://github.com/jenkinsci/google-storage-plugin/blob/develop/README.md).
```
Comment (craigdbarber) [inline gke/README.md:56]: Could you please also add the OAuth plugin to this list: https://github.com/jenkinsci/google-oauth-plugin/blob/develop/README.md

File: gke/README.md [hunk 4]
```diff
@@ -51,8 +51,34 @@ The following describes the setup process for this project:
...
+    gcloud projects add-iam-policy-binding $PROJECT \
+    --member serviceAccount:$SA_EMAIL \
+    --role ‘roles/containeranalysis.admin’
+    gcloud projects add-iam-policy-binding $PROJECT \
+    --member serviceAccount:$SA_EMAIL \
+    --role ‘roles/storage.objectAdmin'
+    gcloud iam service-accounts keys create ./kaniko-secret.json --iam-account $SA_EMAIL
+    ```
+
+1. Add the new credential to your jenkins project:
```
Comment (craigdbarber) [inline gke/README.md]: The kaniko secret doesn't need to be added to the Jenkins cred store, as it's only referenced via the kaniko-pod.yaml which pulls is directly from the k8s secret store.

File: gke/README.md (hunk 3 above)
Comment (donmccasland) [reply gke/README.md:56]: done.

**Review by craigdbarber — APPROVED** [review]
LGTM.
Thanks very much for the contribution!

**Pull request merged**